services/fee-estimator/jest.config.functional.js

```diff
@@ -0,0 +1,13 @@
+// For a detailed explanation regarding each configuration property, visit:
+// https://jestjs.io/docs/en/configuration.html
+
+module.exports = {
+	coverageDirectory: 'test/coverage',
```

Comment:
By adding verbose we will be able to debug better.

```suggestion
        verbose: true,
	coverageDirectory: 'test/coverage',
```